### PR TITLE
Add Attachment renderer to view and edit model attachments

### PR DESCRIPTION
**Pull request by Meroje**

### CODE REVIEW

**Comment by DOMEYD** [comment]
:+1:

File: views/attachemnt/inputs.view.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="attachemnt">
```
Comment (jguyomard) [inline views/attachemnt/inputs.view.php:1]: Typo: this file is named `attachemnt` instead if `attachment`

Comment (SwithFr) [reply views/attachemnt/inputs.view.php:1]: of*

Comment (jguyomard) [reply views/attachemnt/inputs.view.php:1]: typo :p

**Comment by Meroje** [comment]
Feel free to reopen this if you ever need it

**Pull request closed without merging**